### DISCUSSION TITLE
Column name Description

### DISCUSSION THREAD

**Discussion by VishanthSurresh** [sibylhe/mmm_stan#11] [Q&A] [answered]
Hi,

Can you please give the descriptions for Control Variables 

Thanks in advance!

**Comment by sibylhe** [comment, marked as answer]
me_ics_all: consumer sentiment index. https://tradingeconomics.com/united-states/consumer-confidence
me_gas_dpg: gas price
st_ct: store count
mrkdn_valadd_edw: markdown (how many percent off) of value-added service
mrkdn_pdm: markdown of products
hldy: retail holiday
seas: month, starting from Feb (prd_1 is Feb). Nov and Dec are broken down to weeks.
